vars/checkReleaseIssues.groovy

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+import jenkins.ReleaseMetricsData
+ /**
+ * Library to check if release issue exists in the component repo.
+ * @param Map args = [:] args A map of the following parameters
+ * @param args.inputManifest <required> - Input manifest file(s) eg: [2.0.0/opensearch-2.0.0.yml, 2.0.0/opensearch-dashboards-2.0.0.yml] .
+ * @param args.action <optional> - Action to be performed. Default is 'check'. Acceptable values are 'check' and 'create'.
+ */
+void call(Map args = [:]) {
+    def inputManifest = args.inputManifest
+    String action = args.action ?: 'check'
+
+    // Parameter check
+    if (!inputManifest || inputManifest.isEmpty()) {
```

Comment:
same `boolean isNullOrEmpty(String str) { return (str == null || str.allWhitespace || str.isEmpty()) }` here

Reply:
This is also an array. Both manifests for OS and OSD can be supplied at the same time. While calling this method in the orchestrator we will only take version as an input and hardcode manifest file paths.